test/e2e/prometheus_test.go

```diff
@@ -4628,10 +4657,18 @@ func testPrometheusCRDValidation(t *testing.T) {
 				return
 			}
 
-			_, err := framework.CreatePrometheusAndWaitUntilReady(context.Background(), ns, prom)
+			prom, err := framework.CreatePrometheusAndWaitUntilReady(context.Background(), ns, prom)
 			if err != nil {
 				t.Fatalf("expected no error but got %v", err)
 			}
+			if test.name == "no-namespace" {
```

Comment:
Hmm we don't need additional testing. We only want to ensure that an object with an empty namespace is accepted by the API.